### PR TITLE
Add class for <hr> tag #191

### PR DESCRIPTION
**Pull request by abdulmannans**
| Q             | A
| ------------- | ---
| New feature?  | ✔️
| Fixed issues | #191 

Test cases Passed

### CODE REVIEW

**Review by vjik — CHANGES_REQUESTED** [review]
Can you add method to Html helper class and info to readme?

**Comment by abdulmannans** [comment]
@vjik Please check and let me know

**Comment by abdulmannans** [comment]
@vjik can you please review this?

File: src/Html.php [hunk 1]
```diff
@@ -1531,6 +1532,14 @@ public static function header(string|Stringable $content = '', array $attributes
         return $content === '' ? $tag : $tag->content($content);
     }
 
+    /**
+     * Generates a {@see Hr} tag.
+     */
+    public static function hr(): Hr
```
Comment (vjik) [inline src/Html.php:1538]:
```suggestion
    public static function hr(array $attributes = []): Hr
```

Comment (vjik) [inline src/Html.php:1538]: Add tests for this method

**Review by vjik — CHANGES_REQUESTED** [review]
Just a little bit left. See suggestions and add line to changelog.

**Review by vjik — CHANGES_REQUESTED** [review]

**Review by vjik — APPROVED** [review]

**Comment by vjik** [comment]
@abdulmannans Thank you!

**Pull request merged**